catkin_tools/jobs/cmake.py

```diff
@@ -340,9 +353,24 @@ def create_cmake_clean_job(
     build_space = context.package_build_space(package)
     # Package metadata path
     metadata_path = context.package_metadata_path(package)
+    # Environment dictionary for the job, which will be built
+    # up by the executions in the getenv stage.
+    job_env = dict(os.environ)
 
+    # Create job stages
     stages = []
 
+    # Get environment for job.
+    stages.append(FunctionStage(
+        'getenv',
```

Comment:
Could call this loadenv or something instead; I'm not attached one way or another.